builtin/providers/azurerm/resource_arm_availability_set_test.go

```diff
@@ -204,11 +206,11 @@ func testCheckAzureRMAvailabilitySetDestroy(s *terraform.State) error {
 var testAccAzureRMVAvailabilitySet_basic = `
 resource "azurerm_resource_group" "test" {
     name = "acctestRG-%d"
-    location = "West US"
+    location = "Canada Central"
```

Comment:
Actually, it looks like this commit is supposed to be on a separate PR. Fixing...